test/index.js

```diff
@@ -125,11 +125,29 @@ describe('redux-mock-store', () => {
   it('subscribes to dispatched actions', (done) => {
     const store = mockStore();
     const action = { type: 'ADD_ITEM' };
-    
+
     store.subscribe(() => {
       expect(store.getActions()[0]).toEqual(action);
       done();
     });
     store.dispatch(action);
   });
+
+  it('can unsubscribe subscribers', function (done) {
+    const store = mockStore();
+    const action = { type: 'ADD_ITEM' };
+    const waitForMS = 25;
+    const testWaitsAnotherMS = 25;
+
+    this.timeout(waitForMS + testWaitsAnotherMS);
+    const timeoutId = setTimeout(done, waitForMS);
+
+    const unsubscribe = store.subscribe(() => {
+      clearTimeout(timeoutId);
+      done(new Error('should never be called'));
+    });
+
+    unsubscribe()
+    store.dispatch(action);
```

Comment:
@dmitry-zaets Thanks for having a look. Indeed, there are no expectation except for something *not* to be called. In an async environment, doing it like this seemed like a good solution. You can see that it works by adjusting the implementation - the test will go red with `should never be called`.
If you insist I can change it to use a spy, even though I believe it will end up being the same thing, but different.